### PR TITLE
Added Provider Commands

### PR DESCRIPTION
**Pull request by sangeetha5491**
Provider CRUD commands

## Description
event:provider:get <provider_id>
event:provider:list
event:provider:create
event:provider:update <provider_id>
event:provider:delete <provider_id>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/commands/event/provider/update.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            return true
+          }
+          return `The input provider's label '${input}' contains invalid character (valid characters are letters, numbers, underscores, hyphens, dots, parenthesis, comma, @ and space)"`
+        }
+      }, {
+        name: 'description',
+        message: 'Add a description about the provider. (Optional)',
+        validate (input) {
+          // eslint-disable-next-line no-useless-escape
+          const valid = /[\w\s-_\.\(\)\,\@]{0,255}$/
```
[the remark this answers is not in the document]
Comment (sangeetha5491) [reply src/commands/event/provider/update.js:40]: no. its 255 based on the limitation we have set

File: src/commands/event/provider/update.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+          return `The input provider's description '${input}' contains invalid character (valid characters are letters, numbers, underscores, hyphens, dots, parenthesis, comma, @ and space)"`
+        }
+      }, {
+        name: 'docs_url',
+        message: 'Add a url that contains documentation about the provider. (Optional)'
+      }])
+
+      const providerPayload = {
+        label: response.label,
+        description: response.description || undefined,
```
[the remark this answers is not in the document]
Comment (sangeetha5491) [reply src/commands/event/provider/update.js:53]: yes. empty strings result in an error

[the remark this answers is not in the document]
Comment (moritzraho) [reply src/commands/event/provider/update.js:53]: ok thanks

File: src/commands/event/provider/update.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (moritzraho) [reply src/commands/event/provider/update.js]: oh okok thanks for the details it's just that in console I could add a 500 chars description, but that was for the registration

**Pull request merged**